Sources/TuistKit/Services/ProjectAutomation+ManifestMapper.swift

```diff
@@ -13,6 +13,9 @@ extension ProjectAutomation.Graph {
         let projects = targetsAndDependencies
             .map(\.key.project)
             .reduce(into: [String: ProjectAutomation.Project]()) {
+                guard $0[$1.path.pathString] == nil else {
+                    return
+                }
```

Comment:
```suggestion
        let projects = targetsAndDependencies
            .map(\.key.project)
            .uniqued()
            .reduce(into: [String: ProjectAutomation.Project]()) {
```

I believe this is more expressive